l1-contracts/test/Decoder.t.sol

```diff
@@ -14,6 +14,10 @@ import {DecoderHelper} from "./DecoderHelper.sol";
 contract DecoderTest is Test {
   DecoderHelper internal helper;
 
+  // Prime field order
+  uint256 internal constant P =
+    21888242871839275222246405745257275088548364400416034343698204186575808495617;
```

Comment:
There is a constants library where you can get the value

Reply:
Addressed in [912446e](https://github.com/AztecProtocol/aztec-packages/pull/615/commits/912446ee4b492368d5e32a48912c5501954183b5)